### PR TITLE
Create user

### PR DESCRIPTION
**Pull request by AhmadZain01**

### CODE REVIEW

**Comment by muneeb-ahmad98** [comment]
its good.

**Comment by tauqeer-ahmad** [comment]
@AhmadZain01  did we added `developmet.rb` required updates for devise to mailer configuration.

File: Gemfile [hunk 1]
```diff
@@ -22,9 +23,11 @@ gem 'coffee-rails', '~> 4.2'
 gem 'turbolinks', '~> 5'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 gem 'jbuilder', '~> 2.5'
-gem 'jquery-rails'
-gem 'bootstrap-sass', '~> 3.4', '>= 3.4.1'
+gem 'twitter-bootstrap-rails'
+ gem 'devise-bootstrap-views'
```
Comment (tauqeer-ahmad) [inline Gemfile:27]: fix alignment issue.

File: app/views/budgets/_income_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<div class='row'>
+  <div class='col-xs-12'>
+    <%= form_for(@budget, :html => {class: "form-horizontal", role:"form"})do |f| %>
```
Comment (tauqeer-ahmad) [inline app/views/budgets/_income_form.html.erb:3]: fix code alignment issue.

Comment (tauqeer-ahmad) [inline app/views/budgets/_income_form.html.erb:3]: use notation html: {..}

File: app/views/budgets/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    <tr>
+      <th scope="col">Income</th>
+      <th scope="col">Expense</th>
+      <th scope="col">Edit</th>
+      <th scope="col">Delete</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @budgets.each do |budget| %>
+    <tr>
```
Comment (tauqeer-ahmad) [inline app/views/budgets/index.html.erb:18]: fix code alignment

File: app/views/budgets/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      <td><%= budget.income %></td>
+      <td><%= budget.expense %> </td>
+      <td> <%= link_to "Edit  budget", edit_budget_path(budget) %></td>
```
Comment (tauqeer-ahmad) [inline app/views/budgets/index.html.erb:21]: remove extra space from "Edit  budget"

File: config/environments/development.rb [hunk 5]
```diff
@@ -41,14 +38,29 @@
...
-  config.active_record.verbose_query_logs = true
-
   # Debug mode disables concatenation and preprocessing of assets.
   # This option may cause significant delays in view rendering with a large
   # number of complex assets.
   config.assets.debug = true
 
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+  address: "smtp.gmail.com",
```
Comment (tauqeer-ahmad) [inline config/environments/development.rb:48]: fix code alignment.

File: app/views/budgets/_income_form.html.erb [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    <div class="form-group">
+      <div class="control-label col-sm-8">
+        <%= f.text_field :income , class:"form-control", placeholder: "Income", autofocus: true %>
+      </div>
+    </div>
+    <div class="form-group">
+      <div class="control-label col-sm-8">
+        <%= f.text_field :expense , class:"form-control", placeholder: "Title of the Article", autofocus: true %>
```
Comment (tauqeer-ahmad) [inline app/views/budgets/_income_form.html.erb:11]: add space after `class:`

**Pull request merged**